includes/blocks/class-sensei-block-quiz-question.php

```diff
@@ -22,39 +22,18 @@ public function __construct() {
 		Sensei_Blocks::register_sensei_block(
 			'sensei-lms/quiz-question',
 			[
-				'render_callback' => '__return_empty_string',
+				'render_callback' => [ $this, 'render_quiz_question' ],
 			],
 			Sensei()->assets->src_path( 'blocks/quiz/question-block' )
 		);
-		Sensei_Blocks::register_sensei_block(
-			'sensei-lms/question-description',
-			[
-				'render_callback' => '__return_empty_string',
```

Comment:
Isn't the issue related to these `render_callback` only? I suspect it was expected only in the `sensei-lms/quiz-question` based on the old code.

Reply:
Let me take a closer look. In the interest of getting this shipped quickly (hopefully this week), my original plan was to revert and then fix the translation issue at some later point in time. But let's see.

Reply:
So I tried calling the following instead for just the question description (which I think is what you meant?) but it didn't work:

```
Sensei_Blocks::register_sensei_block(
	'sensei-lms/question-description',
	[
		'render_callback' => [ $this, 'render_quiz_question' ],
	],
	Sensei()->assets->src_path( 'blocks/quiz/question-description-block' )
);
```

Reply:
I mean not having the `render_callback` in this case. Something like this:

```
Sensei_Blocks::register_sensei_block(
	'sensei-lms/question-description',
	[],
	Sensei()->assets->src_path( 'blocks/quiz/question-description-block' )
);
```

And I think we probably should do the same with the `sensei-lms/question-answers` since the original purpose was only to register in the backend, but not customize the `render_callback`.

Reply:
Got it. Updated here https://github.com/Automattic/sensei/pull/7669/commits/d1399a935f7c0ec0338091ff4dc322b3c7b8065e.